targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F469xI/TARGET_SDP_K1/PinNames.h

```diff
@@ -306,22 +306,25 @@ typedef enum {
 	// Debug pins
 	DEBUG_GPIO0	= PG_6,
     // Generic signals namings
-    LED1        = PK_4,	// Status LED
-    LED2        = PK_7, // Red LED
-    LED3        = PK_6, // Orange LED
-    LED4        = PK_5, // Green LED
-    AWAKE        = PK_3,	
+    LED1        = PK_7, // Red LED
+    LED2        = PK_6, // Orange LED
+    LED3        = PK_5, // Green LED
+    LED_RED		= LED1,
+	LED_ORANGE	= LED2,
```

Comment:
Please fix alignment in this file (LED_ORANGE aligned with LED_RED and others for instance)

Reply:
Thanks, @0xc0170. 
https://github.com/ARMmbed/mbed-os/pull/10471/commits/e0faeb2b65353e74d5b171ee2b595d68816e18b0